### PR TITLE
PEP8 rolling through

### PR DESCRIPTION
**Pull request by betatim**
Nothing to see here, just some PEP8'ing.

### CODE REVIEW

File: papermill/tests/test_execute.py [hunk 1]
```diff
@@ -2,22 +2,25 @@
...
 from . import get_notebook_path, RedirectOutput
 
-python_2 = sys.version_info[0] == 2
-PYTHON = 'python2' if python_2 else 'python3'
-execute_notebook = partial(execute_notebook, kernel_name=PYTHON)
+
+if six.PY2:
+    execute_notebook = partial(execute_notebook, kernel_name='python2')
+else:
+    execute_notebook = partial(execute_notebook, kernel_name='python3')
```
Comment (MSeal) [inline papermill/tests/test_execute.py]: simpler as `execute_notebook = partial(execute_notebook, kernel_name='python3' if six.PY2 else 'python2')`

File: papermill/tests/test_parameterize.py [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 import unittest
 
+import six
+
 from ..api import read_notebook
 from ..execute import _parameterize_notebook
 from . import get_notebook_path
 
-PYTHON = 'python2' if sys.version_info[0] == 2 else 'python3'
+if six.PY2:
```
Comment (MSeal) [inline papermill/tests/test_parameterize.py]: nbd but I think the single line version reads better `PYTHON = 'python2' if six.PY2 else 'python3'`

Comment (betatim) [reply papermill/tests/test_parameterize.py]: 👍

File: papermill/tests/test_execute.py (hunk 1 above)
Comment (betatim) [reply papermill/tests/test_execute.py]: Can we split the difference on this one? I wouldn't ever expect a ternary inside an argument like that and it makes the line super long (as in >79chars which is my religion ;) )

Comment (MSeal) [reply papermill/tests/test_execute.py]: That 80 char limit is super irritating in modern coding environments (especially with 4 space python :P). If you really don't like the ternary then at least keep it a bit more DRY and make a kernal_name variable which captures the six logic and only repeat `execute_notebook = partial(execute_notebook, kernel_name=kernel_name)` once

Comment (MSeal) [reply papermill/tests/test_execute.py]: Sound alright? If you strong disagree I'm still ok with merging as it's better than it was regardless.

Comment (betatim) [reply papermill/tests/test_execute.py]: Done.

Next you are going to tell me some of your friends use tabs :p

**Pull request merged**